geoips_clavrx/plugins/modules/algorithms/cloud/cldHeightBase.py

```diff
@@ -44,7 +44,7 @@ def call(xobj, output_data_range=None):
     cldHeightBase_min = output_data_range[0]
     cldHeightBase_max = output_data_range[1]
     cldHeightBase = (
-        xobj["cld_height_base_acha"].to_masked_array() / 1000.0
+        xobj["cld_height_base"].to_masked_array() / 1000.0
```

Comment:
This should likely be updated at some point to actually use the variables from the product YAMLs. I'm ok leaving it for now.

Reply:
Yeah, I find this to be kind of annoying, but it's fine for now. In general, I'd like to rework things to make better use of the plugins.